.github/workflows/test-docs.yml

```diff
@@ -14,9 +14,4 @@ jobs:
         with:
           node-version: 16
           cache: yarn
-      - run: ./scripts/docs-build.sh
-  markdown-link-check:
```

Comment:
Remove markdown link checker.  No longer necessary now that we fail on missing links in docs right?